### PR TITLE
landing page design

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

File: my-app/src/App.js [hunk 1]
```diff
@@ -1,41 +1,29 @@
...
+
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/history" element={<History />} />
+        <Route path="/liked" element={<Liked />} />
+        <Route path="/playlist" element={<Playlist />} />
+        <Route path="/watchLater" element={<WatchLater />} />
+      </Routes>
+
+      <Footer />
```
Comment (therajatg) [inline my-app/src/App.js]: you can have all your routes in a different file.

File: my-app/src/frontend/components/footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <div className="social_content">
+        <div className="social_upper_content">Connect with us</div>
+        <div className="social_logo">
+          <FaTwitter className="twitter" />
+          <GrFacebookOption className="facebook" />
+        </div>
+      </div>
+    </div>
+  );
+}
```
Comment (therajatg) [inline my-app/src/frontend/components/footer/Footer.js]: I like the classnames you used.

**Review by therajatg — APPROVED** [review]

File: my-app/src/App.js (hunk 1 above)
Comment (parulgupta2001) [reply my-app/src/App.js]: I think there is no need of this.

File: src/frontend/pages/home/Home.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import "./home.css";
+import "../../../App.css";
+
+export function Home() {
+  const [videoData, setVideoData] = useState([]);
+
+  useEffect(() => {
+    axios.get("/api/videos").then((response) => {
+      setVideoData(response.data.videos);
+      console.log(response.data.videos);
```
Comment (junaidshaikh-js) [inline src/frontend/pages/home/Home.js:16]: remove console logs

**Review by junaidshaikh-js — APPROVED** [review]
Try to provide a good description of the changes/features you worked on in particular PR. Also, try to raise PR with fewer file changes.

File: src/frontend/pages/home/Home.js (hunk 3 above)
Comment (parulgupta2001) [reply src/frontend/pages/home/Home.js:16]: ok

**Pull request merged**